src/mongocxx/test/collection.cpp

```diff
@@ -11,15 +12,20 @@
 #include <mongocxx/client.hpp>
 #include <mongocxx/pipeline.hpp>
 
+using namespace bsoncxx::builder::stream;
 using namespace mongocxx;
 
+namespace {
+const auto kEmptyDoc = document{} << finalize;
+}  // namespace
+
 TEST_CASE("collection renaming", "[collection]") {
     client mongodb_client{uri{}};
     database db = mongodb_client["test"];
 
     std::string collname{"mongo_cxx_driver"};
     collection coll = db[collname];
-    coll.insert_one({});  // Ensure that the collection exists.
+    coll.insert_one(kEmptyDoc.view());  // Ensure that the collection exists.
```

Comment:
Hmm. This would be a sad consequence of view_or_value if we can't do the `insert_one({})`, `find_one({})` etc. anymore. I think we should try to find a way around this. Can we static assert that `View`s are default constructible in the template, than add a default constructor for v_or_v that just uses `View`'s default?